docs/source/api/v4/multiple_server_capabilities.rst

```diff
@@ -0,0 +1,83 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-multiple_server_capabilities:
+
+********************************
+``multiple_server_capabilities``
+********************************
+
+.. versionadded:: 4.1
+
+``PUT``
+========
+Associates a list of :term:`Server Capability` to a server.
```

Comment:
To be clear, what this actually does is replace all of the Server Capabilities assigned to a server with the ones specified. It doesn't just associate the ones you send with the server, it also gets rid of all other such associations.

Which is appropriate for a PUT request, but the docs could be a bit more clear on that point.

Reply:
Sure.